partner_industry_secondary/migrations/11.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def fill_res_partner_industry(cr):
+    cr.execute(
+        """
+        INSERT INTO res_partner_industry (name, active, create_uid,
+            create_date, write_uid, write_date, parent_id, %s)
+        SELECT name, TRUE as active, create_uid, create_date, write_uid,
+            write_date, parent_id, id as %s
```

Comment:
No need here to put the `as %s`, as field selection is done per order

Reply:
Remove `parent_id` from this sentence, or you will have foreign key errors

Reply:
I don't have `parent_id` errors because they are amended in line 21

Reply:
> No need here to put the as %s, as field selection is done per order

But also no need to take it out

Reply:
> You didn't have previously the problem as the fields were created by hand (with the add_fields openupgradelib method)

See! That's why I shouldn't have deleted the `add_fields` 😜

Reply:
hahaha